pkg/sdk/logging/api/v1beta1/logging_types.go

```diff
@@ -206,7 +206,7 @@ const (
 	DefaultFluentdDrainWatchImageTag              = "v0.2.3"
 	DefaultFluentdDrainPauseImageRepository       = "k8s.gcr.io/pause"
 	DefaultFluentdDrainPauseImageTag              = "3.2"
-	DefaultFluentdVolumeModeImageRepository       = "busybox"
+	DefaultFluentdVolumeModeImageRepository       = "docker.io/busybox"
```

Comment:
```suggestion
	DefaultFluentdVolumeModeImageRepository       = "docker.io/library/busybox"
```

official Docker Hub image repositories are within the `library` namespace

Reply:
Thanks for the headsup!